reference/orders.v2.oas2.yml

```diff
@@ -4412,13 +4620,229 @@ components:
         - type: object
           properties:
             products:
-              type: array
-              items:
-                anyOf:
-                  - $ref: '#/components/schemas/orderCustomProduct_Post'
-                  - $ref: '#/components/schemas/orderCatalogProduct_Post'
+              $ref: '#/components/schemas/orderProduct_Post'
```

Comment:
we may need to update the line "description: Products and Billing address only required for POST operation." to mention about either "consignments" or "products" field is required

Reply:
https://github.com/bigcommerce/api-specs/pull/315/files#r623521605

Reply:
👍